.github/workflows/cd.yml

```diff
@@ -0,0 +1,38 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - synchronize
+      - opened
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+
+  plugin_zip:
+    name: Create archive
+    runs-on: ubuntu-20.04
+    env:
+      OUTPUT: High DPI Continuous.zip
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          show-progress: false
+      - name: Create archive
+        uses: thedoctor0/zip-release@0.7.5
+        with:
+          type: 'zip'
+          filename: 'High DPI Continuous.zip'
+          exclusions: '*.git*'
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.OUTPUT }}
+          path: ${{ env.OUTPUT }}
```

Comment:
Make a continuous release.

```suggestion


  continuous_release:
    name: Upload
    if: github.event_name == 'push'
    runs-on: ubuntu-latest
    needs:
      - plugin_zip
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      OUTPUT: High DPI Continuous.zip
    steps:
      - uses: actions/checkout@v4
        with:
          show-progress: false
      - name: Install github-release
        run: |
          go install github.com/github-release/github-release@latest
          echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
      - name: Set environment variables
        run: |
          echo "GITHUB_USER=$( echo ${{ github.repository }} | cut -d/ -f1 )" >> $GITHUB_ENV
          echo "GITHUB_REPO=$( echo ${{ github.repository }} | cut -d/ -f2 )" >> $GITHUB_ENV
      - name: Move/Create continuous tag
        run: |
          git tag --force continuous ${{ github.sha }}
          git push --tags --force
      - name: Setup continuous release
        run: |
          DESCRIPTION="Triggered on $(date -u '+%Y/%m/%d, %H:%M') UTC by commit ${{ github.sha }} (@${{ github.actor }})

          This is an automated build of the latest source. It may be unstable or even crash, corrupt your save or eat your kitten. Use with caution!

          https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
          if ! github-release info -t continuous > /dev/null 2>&1; then
            github-release release \
              --tag continuous \
              --name "Continuous" \
              --description "$DESCRIPTION" \
              --pre-release
          else
            github-release edit \
              --tag continuous \
              --name "Continuous Build" \
              --description "$DESCRIPTION" \
              --pre-release
          fi
      - name: Download Artifacts
        uses: actions/download-artifact@v4
        with:
          path: ${{ github.workspace }} # This will download all files to e.g `./High DPI Continuous.zip/High DPI Continuous.zip`
      - name: Add ${{ env.OUTPUT }} to release tag
        continue-on-error: true
        run: |
          github-release upload \
            --tag continuous \
            --replace \
            --name ${{ env.OUTPUT }} \
            --file ${{ env.OUTPUT }}/${{ env.OUTPUT }}

```